.github/workflows/docker.yml

```diff
@@ -1,16 +1,18 @@
 name: Build Base Docker Image
 
-# Triggered only if environment.yml in main changes
+# Triggered only if environment_meson.yml in main changes
 # Pushes a Docker image with all dev dependencies to Docker Hub
 on:
   push:
     branches:
       - main
     paths:
+      # will keep both until there is a reason to get rid of scipy-dev entirely
+      - 'environment_meson.yml'
```

Comment:
so no need for this comment/change, let's get rid of `environment_meson.yml` instead. that also addresses Tania's other comments about the environment name change.